support-frontend/assets/pages/supporter-plus-landing/twoStepTest/checkoutScaffold.tsx

```diff
@@ -130,7 +130,16 @@ export function SupporterPlusCheckoutScaffold({
 		isUsEoy2023CampaignEnabled && countryGroupId === 'UnitedStates';
 
 	const heading = showUsEoy2023Content ? (
-		<LandingPageHeading heading="Make a year-end gift to the Guardian" />
+		<LandingPageHeading
+			heading={
+				<>
+					Make a<br />
+					year-end gift
+					<br />
+					to the Guardian
+				</>
```

Comment:
Does this work across all breakpoints?

Reply:
It does indeed in the sense of that the left column is a fixed width across all breakpoints, and eventually just disappears.

## Wide
![Screenshot 2023-11-08 at 10 24 44](https://github.com/guardian/support-frontend/assets/31692/75c18198-0adc-4680-b527-12901aabfd56)

## Widish
![Screenshot 2023-11-08 at 10 24 55](https://github.com/guardian/support-frontend/assets/31692/350dd818-dbc8-42eb-9e7d-ad8866ea9b4e)

## Narrow + gone
![Screenshot 2023-11-08 at 10 25 03](https://github.com/guardian/support-frontend/assets/31692/9e4eca15-7310-4f3d-b0eb-f36d5114f08f)